.changeset/bright-cars-explain.md

```diff
@@ -0,0 +1,5 @@
+---
+'gitbook': patch
+---
+
+Add automatic color contrast in site header, restyle search button
```

Comment:
We don't need a changeset I think.

Reply:
We should have changeset for everything. It acts as a changelog for our public content output.

Reply:
OK I didn't know. So we should deploy only if changeset is present (is it the case?).